例外処理
try{
	通常事項される文
} catch() {
	例外発生時に実行される文
}
